src/OpenRoad.cc

```diff
@@ -438,28 +438,39 @@ void OpenRoad::writeCdl(const char* outFilename,
 }
 
 void OpenRoad::readDb(const char* filename)
+{
+  std::ifstream stream;
+  stream.open(filename, std::ios::binary);
+  try {
+    readDb(stream);
+  } catch (const std::ios_base::failure& f) {
+    logger_->error(ORD, 54, "odb file {} is invalid: {}", filename, f.what());
```

Comment:
doesn't this destroy the stack trace information? wouldn't that be useful in logs sometimes?

Reply:
No change in functionality.

Reply:
but catching and rethrowing looses the stack trace. is that desireable/intentional?

Reply:
It caught and threw before this PR so there is no change.  This isn't a crash so there is no stack trace to see.